dnf/cli/output.py

```diff
@@ -1180,7 +1180,8 @@ def format_line(group):
                     lines.append(format_line(group))
                 pkglist_lines.append((action, lines))
         # show skipped conflicting packages
-        if not self.conf.best and self.base._goal.actions & forward_actions:
+        if ((not self.conf.best or not self.conf.strict) and not self.conf.skip_broken) \
```

Comment:
Another condition was added to keep existing behavior with `--skip-broken` option. When using this option, no problem and no skipped package is reported in the output. Maybe the list of skipped packages could be useful in this context, but current condition is connected also with reporting the problem in the output which we don't want to in this case. It would need some refactoring...

Reply:
The [`--skip-broken`](https://dnf.readthedocs.io/en/latest/command_ref.html?#skip-broken-option-label) option is just an alias for `strict=false` I think we shouldn't use it directly it should stay only an alias in the [`config`](https://github.com/rpm-software-management/dnf/blob/master/dnf/conf/config.py#L417).

Based on that I think we don't want to keep its existing behavior - because we don't want to keep the existing behavior of `strict=false` (when `best=true`).

Reply:
OK I missed the docs here, thanks, I understand now. I will drop the option and fix the new wanted behavior in related test.